File: Assets/Tests/plugins/UIKitTransformPlugin.swift

class ViewController: UIViewController {
    @IBOutlet weak var view: UIView!
    @IBOutlet weak var label: UILabel!
    @IBOutlet weak var textField: UITextField!
    @IBOutlet weak var imageView: UIImageView!
    @IBOutlet weak var button: UIButton!
    @IBOutlet weak var tableView: UITableView!
    @IBOutlet weak var stackView: UIStackView!
    @IBOutlet weak var scrollView: UIScrollView!
    @IBOutlet weak var aSwitch: UISwitch!

    func method(label: UILabel) -> UIView {
        let view = UIView()
        view.addSubview(view)
        return view
    }

    @IBAction func onTap() {
    }
}

File: Assets/Tests/plugins/UIKitTransformPlugin.kt

internal class ViewController: UIViewController {
    @BindView() lateinit internal var view: View
    @BindView() lateinit internal var label: TextView
    @BindView() lateinit internal var textField: EditText
    @BindView() lateinit internal var imageView: ImageView
    @BindView() lateinit internal var button: Button
    @BindView() lateinit internal var tableView: RecyclerView
    @BindView() lateinit internal var stackView: LinearLayout
    @BindView() lateinit internal var scrollView: ScrollView
    @BindView() lateinit internal var aSwitch: Switch
    
    internal fun method(label: TextView) : View {
        val view = View()
        view.addSubview(view)
        return view
    }
    
    @OnClick() internal fun onTap() {}
}
